src/reducers/breakpoints.js

```diff
@@ -24,13 +24,15 @@ export type BreakpointsState = {
   breakpointsDisabled: false
 };
 
-export const State = makeRecord(
-  ({
-    breakpoints: I.Map(),
-    pendingBreakpoints: restorePendingBreakpoints(),
-    breakpointsDisabled: false
-  }: BreakpointsState)
-);
+export function initialState() {
```

Comment:
Hmm I tried this, but it introduced a few problems. do you mean that the makeRecord should be moved out?